app/views/answer_mailer/question_answered.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Answer#question_answered</h1>
+<%= @question.user_email%> receive answer <%= @answer.body%> by <%= @admin.email%>
+<p>
+  <%= @greeting %>, find me in app/views/answer_mailer/question_answered.html.erb
```

Comment:
it's strange to add such lines in mails like `find me in app/views/answer_mailer/question_answered.html.erb`
what does it mean?